upgrade/upgrade.go

```diff
@@ -230,6 +207,31 @@ func doAPIVersionUpgrade(namespace string, config *restclient.Config, lhClient *
 	return nil
 }
 
+func canUpgrade(config *restclient.Config, namespace string) (bool, error) {
+	lhClient, err := lhclientset.NewForConfig(config)
```

Comment:
Shouldn't this clientset be `lhclientset_v1beta1` if you are checking CRD API v1beta1?

Reply:
Yes. Using old lhclientset_v1beta1 is better.